core/clingutils/CMakeLists.txt

```diff
@@ -46,7 +46,7 @@ foreach(dict ${stldicts})
   string(REPLACE "complex" "root_std_complex.h" header ${header})
   string(REPLACE "multi" "" header ${header})
   ROOT_GENERATE_DICTIONARY(G__std_${dict} ${header} STAGE1 MODULE ${dict}Dict LINKDEF src/${dict}Linkdef.h DEPENDENCIES  Core)
-  ROOT_LINKER_LIBRARY(${dict}Dict G__std_${dict}.cxx DEPENDENCIES  Core)
+  ROOT_LINKER_LIBRARY(std_${dict} G__std_${dict}.cxx DEPENDENCIES  Core)
```

Comment:
Yes, see the commit about fixing the naming which states that this prevents dependency linking between compiling and generating (and my comment above).